index.html

```diff
@@ -8,21 +9,23 @@
   </head>
   <body>
     <aside>
-      <div> IdeaBox
-      </div>
-      <div class="filter-section"> Filter Starred Ideas
-        <button> Show Starred Ideas </button>
+      <h1> IdeaBox </h1>
+      <hr>
+      <div class="filter-section">
+        <h3>Filter Starred Ideas</h3>
+        <button class ='button aside'> Show Starred Ideas </button>
       </div>
+      <hr>
     </aside>
     <main>
       <section>
         <div class="idea-input-form">
-        <p> Title </p>
-        <input type="text">
-        <p> Body </p>
-        <input type="text"> </input>
-        <button> Save </button>
-        <input type="text" placeholder="Search ideas"> </input>
+          <p> Title </p>
+          <input type="text">
+          <p> Body </p>
+          <input type="text" class='input large'> </input>
+          <button> Save </button>
+          <input type="text" placeholder="Search ideas" class='input'> </input>
```

Comment:
Good, but I think you're right, I think that "body" field is ultimately gonna need to be a textarea.